core/modules/color/css/color.admin.css

```diff
@@ -28,16 +28,32 @@ input[type=color] {
   margin-top: .5em;
   margin-bottom: .5em;
   clear: both;
-  vertical-align: middle;
 }
 
 /* Color inputs */
```

Comment:
This no longer sits directly above `#system-theme-settings input[type=color] {` like it used to. It needs to be moved below the newly-added code.

Reply:
Sorry for being thick, but I don't understand what you mean @BWPanda. Can you explain further please?

Reply:
The `Color inputs` comment originally sat right above the code to which it referred, not it's separated by the new code. So it should be moved down to sit above the `#system-theme-settings input[type=color] {` line again.

Reply:
Sorry for taking this long ...I gotcha now. Fixing...

Reply:
Fixed. Thanks for pointing this out @BWPanda 🙏